lotus/client.go

```diff
@@ -35,7 +33,8 @@ func New(maddr ma.Multiaddr, authToken string) (*apistruct.FullNodeStruct, func(
 	closer, err := jsonrpc.NewMergeClient("ws://"+addr+"/rpc/v0", "Filecoin",
 		[]interface{}{
 			&api.Internal,
-		}, headers, jsonrpc.WithReconnect(true, time.Second*3, 0))
```

Comment:
The reconnect thing is still enabled. I did a PR some months ago in Lotus with this feature.
https://github.com/filecoin-project/lotus/pull/1155

So, it's baked in now.